base/windows/YesNoMenu.ts

```diff
@@ -97,4 +112,8 @@ export class YesNoMenu {
     destroy() {
         this.menu.destroy();
     }
+
+    get_confirm_sfx() {
```

Comment:
Since it's a public property, you can remove remove this `get_confirm_sfx` method.